target/linux/ramips/mt7621/base-files/etc/board.d/02_network

```diff
@@ -44,6 +44,9 @@ ramips_setup_interfaces()
 	mikrotik,routerboard-750gr3)
 		ucidef_set_interfaces_lan_wan "lan2 lan3 lan4 lan5" "wan"
 		;;
+	mikrotik,routerboard-760igs)
+		ucidef_set_interfaces_lan_wan "eth2 eth3 eth4 eth5" "eth1 sfp1"
```

Comment:
So, eth1 is a wan port?

Reply:
Take a look at https://i.mt.lv/cdn/rb_images/1539_l.jpg and you will see the eth1 and sfp are both "internet" on the front of the device.  But I think naming then lanX fits better since the front of the device calls the ports LAN too, just like the rb750gr3

Reply:
It shares the "Internet" labelling with the SFP cage, see https://i.mt.lv/cdn/rb_images/1539_l.jpg

Reply:
Convinced.